corehq/motech/fhir/views.py

```diff
@@ -84,7 +84,7 @@ def search_view(request, domain, fhir_version_name, resource_type):
     fhir_version = _get_fhir_version(fhir_version_name)
     if not fhir_version:
         return JsonResponse(status=400, data={'message': "Unsupported FHIR version"})
-    resource_id = request.GET.get('_id')
+    resource_id = request.GET.get('_id', request.GET.get('patient_id'))
```

Comment:
maybe a nit:

I usually find that it's better to do `a.get('x', None) or a.get('y')`. 'y' will still be used if 'x' is present but empty. (unless you specifically want to allow and empty value.